tests/flows/validator_set_sig.go

```diff
@@ -0,0 +1,157 @@
+	// Construct ValidatorSetSig message with mock ERC20 as the target contract
+	// Create a OffChainWarp Message using the ValidatorSetSig message to be signed by the validator subnet
+	// Confirm the event is emitted
+
+	cChainInfo := network.GetPrimaryNetworkInfo()
+	contractSubnet, validatorSubnet := utils.GetTwoSubnets(network)
+	_, fundedKey := network.GetFundedAccountInfo()
+
+	ctx := context.Background()
+
+	// Deploy a ValidatorSetSigContract to contract subnet
+	validatorSetSigContractAddress, validatorSetSig := utils.DeployValidatorSetSig(
+		ctx,
+		fundedKey,
+		contractSubnet,
+		validatorSubnet,
+	)
+
+	// Deploy a mock ERC20 contract to contract subnet
+	exampleERC20ContractAddress, exampleErc20 := utils.DeployExampleERC20(
+		ctx,
+		fundedKey,
+		contractSubnet,
+	)
+	erc20ABI, err := exampleerc20.ExampleERC20MetaData.GetAbi()
+	Expect(err).Should(BeNil())
+
+	// Confirm that the validatorContract has a balance of 0 to begin with
+	startingBalance, err := exampleErc20.BalanceOf(&bind.CallOpts{}, validatorSetSigContractAddress)
+	Expect(err).Should(BeNil())
+	Expect(startingBalance.Cmp(big.NewInt(0))).Should(BeZero())
+
+	// Construct a ValidatorSetSig message with mock ERC20 as the target contract
+	// and mint 100 tokens as the TxPayload
+	callData, err := erc20ABI.Pack("mint", big.NewInt(100))
+	Expect(err).Should(BeNil())
+
+	vssMessage1 := validatorsetsig.ValidatorSetSigMessage{
+		ValidatorSetSigAddress: validatorSetSigContractAddress,
+		TargetContractAddress:  exampleERC20ContractAddress,
+		TargetBlockchainID:     contractSubnet.BlockchainID,
+		Nonce:                  big.NewInt(1),
+		Payload:                callData,
+	}
+
+	// Construct a second ValidatorSetSig message with mock ERC20 as the target contract
+	// and mint 50 tokens as the TxPayload
+	callData2, err := erc20ABI.Pack("mint", big.NewInt(50))
+	Expect(err).Should(BeNil())
+	vssMessage2 := validatorsetsig.ValidatorSetSigMessage{
+		ValidatorSetSigAddress: validatorSetSigContractAddress,
+		TargetContractAddress:  exampleERC20ContractAddress,
+		TargetBlockchainID:     contractSubnet.BlockchainID,
+		Nonce:                  big.NewInt(2),
+		Payload:                callData2,
+	}
+
+	networkID := network.GetNetworkID()
+
+	noMessageWarpEnabledChainConfig := utils.GetWarpEnabledChainConfig(nil)
+	// Create chain config file with off chain validatorsetsig message
+	offchainMessages, warpEnabledChainConfigWithMsg := utils.InitOffChainMessageChainConfigValidatorSetSig(
+		networkID,
+		validatorSubnet,
+		validatorSetSigContractAddress,
+		[]validatorsetsig.ValidatorSetSigMessage{vssMessage1, vssMessage2})
+
+	// Create chain config with off chain messages
+	chainConfigs := make(map[string]string)
+	utils.SetChainConfig(chainConfigs, cChainInfo, noMessageWarpEnabledChainConfig)
+	utils.SetChainConfig(chainConfigs, validatorSubnet, warpEnabledChainConfigWithMsg)
+	utils.SetChainConfig(chainConfigs, contractSubnet, noMessageWarpEnabledChainConfig)
```

Comment:
Is it necessary to set the chain config for chains that do not sign off-chain Warp messages?

Reply:
No it's not. Good catch =)